src/js/components/BreadcrumbComponent.jsx

```diff
@@ -0,0 +1,87 @@
+var React = require("react/addons");
+var Link = require("react-router").Link;
+
+var ViewHelper = require("../helpers/ViewHelper");
+
+var BreadcrumbComponent = React.createClass({
+  displayName: "BreadcrumbComponent",
+
+  propTypes: {
+    app: React.PropTypes.string,
+    group: React.PropTypes.string,
+    task: React.PropTypes.string
+  },
+
+  getGroupLinks: function () {
+    var group = this.props.group;
+    if (group == null) {
+      return null;
+    }
+
+    var pathParts = group.split("/").slice(0, -1);
```

Comment:
Just for fun and maybe speed (can't proof), this would be the lazy way:
```javascript
    var id = "";
    
    var pathPartsSeq = lazy(group).split("/").initial();

    return pathPartsSeq
      .map((name, i) => {
        id += pathPartsSeq.skip(i).take(1).value();
        return (
          <li key={id}>
            <Link to="group" params={{groupId: encodeURIComponent(id)}}>
              {name}
            </Link>
          </li>
        );
      })
      .rest()
      .value();
```

Done with an Einbecker Bock.